bin/cli

```diff
@@ -1,23 +1,38 @@
 #!/usr/bin/env node
 
-// Perform rewrites to the runtime environment variables based on the contents
-// of the process.env.REWRITE_ENV if it exists. This is done here as it is the
-// entrypoint for the entire application.
-require('env-rewrite').rewrite();
-
 /**
  * Module dependencies.
  */
 
 const program = require('commander');
 const pkg = require('../package.json');
+const dotenv = require('dotenv');
```

Comment:
It looks like the `dotenv` documentation considers `export` optional in the `.env` files that `dotenv` sources, if it's preferable from a devops side to use local environment variable files for development of the format:
```
NAME=value
```
rather than 
```
export NAME=value
```
then all this looks good. Approving.

(Parotting back what I'm hearing just to confirm that I understand.)